libs/dh/shared/domain/src/lib/generated/v1/model/metering-point-cim-dto.ts

```diff
@@ -54,7 +54,7 @@ export interface MeteringPointCimDto {
     darReference?: string | null;
     capacity?: number | null;
     assetType?: AssetType;
-    settlementMethod?: SettlementMethod;
+    settlementMethod?: SettlementMethod | null;
```

Comment:
It was added manually. There is an inconsistency with the auto-generated interface vs the interface from the web API.

I will remove this manual addition and change the test spec to handle `undefined` instead of null.

Reply:
If there's an inconsistency in the auto-generated files, then we should look at the tool that is used for generating the types.
Manually changes to auto-generated files is better to be avoided.